packages/core/src/storage/migration/updates/0.1-0.2/mediation.ts

```diff
@@ -1,6 +1,6 @@
-import type { V0_1ToV0_2UpdateConfig } from '.'
 import type { Agent } from '../../../../agent/Agent'
 import type { MediationRecord } from '../../../../modules/routing'
+import type { V0_1ToV0_2UpdateConfig } from './index'
```

Comment:
Does this type come from the index file?

Reply:
Yes, do you think we should extract it to a seperate file?